app/indexers/concerns/hyrax/location_indexer_decorator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# OVERRRIDE HYRAX v5.0.1 to turn loc into a string for the location lookup,
```

Comment:
```suggestion
# OVERRIDE HYRAX v5.0.1 to turn loc into a string for the location lookup,
```

Being a little picky, but often I do search for the `OVERRIDE` comment so this might get missed

Reply:
no totally, you caught my typo! thanks!!

Reply:
@kirkkwang would you mind re approving?